.travis.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+group: stable
+dist: trusty
+
+language: python
+python: "2.7"
+
+branches:
+  only:
+    - develop
+    - master
+
+env:
+  - ANSIBLE_INSTALL_VERSION=2.0.2.0
+  - ANSIBLE_INSTALL_VERSION=2.1.5.0
+  - ANSIBLE_INSTALL_VERSION=2.2.3.0
+  - ANSIBLE_INSTALL_VERSION=2.3.1.0
```

Comment:
Can we have the latest versions here?

```
  - ANSIBLE_INSTALL_VERSION=2.0.2.0
  - ANSIBLE_INSTALL_VERSION=2.1.6.0
  - ANSIBLE_INSTALL_VERSION=2.2.3.0
  - ANSIBLE_INSTALL_VERSION=2.3.2.0
  - ANSIBLE_INSTALL_VERSION=2.4.1.0
```

Reply:
Done.  We should probably update this is scaffold as well (since that's from where I got the file).